packages/expo-video/ios/NowPlayingArtworkCache.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+import MediaPlayer
+import Foundation
+
+// If we ever want to use this class for something else we should probably make these configurable through the constructor
+// I couldn't find any information on recommended artwork resolution, but this seems like a reasonable value considering how it's displayed ~@behenate
+private let maxImageSize = CGSize(width: 1024, height: 1024)
+private let cacheCompressionQuality: CGFloat = 0.8
+private let cacheCountLimit = 20
+
+public class NowPlayingArtworkCache {
```

Comment:
Generally speaking, I don't think adding that functionality to `expo-video` is necessary. We probably should use `expo-image` or `expo-filesystem` to somehow provide artwork. However, for now, it might be okay to merge it as it is 🤔